tests/dummy/app/routes/query/route.js

```diff
@@ -24,6 +24,7 @@ export default Route.extend({
     let token = heimdall.start('ember-data');
     return this.get('store').query(modelName, params)
       .then((records) => {
+        records.toArray();
```

Comment:
@runspired do you mind adding a comment explaining this line forces the materialization so we don't forget in the future?